vsphere/datadog_checks/vsphere/event.py

```diff
@@ -109,12 +114,14 @@ def transform_vmbeinghotmigratedevent(self):
         self.payload["msg_text"] += "\n".join(changes)
 
         self.payload['host'] = self.raw_event.vm.name
-        self.payload['tags'] = [
-            'vsphere_host:{}'.format(ensure_unicode(pre_host)),
-            'vsphere_host:{}'.format(ensure_unicode(new_host)),
-            'vsphere_datacenter:{}'.format(ensure_unicode(pre_dc)),
-            'vsphere_datacenter:{}'.format(ensure_unicode(new_dc)),
-        ]
+        self.payload['tags'].extend(
```

Comment:
It seems that extending here will modify the initial list of `tags` passed to the constructor `VSphereEvent(event, self.event_config[i_key], tags)`. Should not have consequences here, but it's better to avoid this kind of side effect.

Reply:
Good catch. You're sure it doesn't have an effect?

Reply:
Not sure what you mean by having an effect :)
I just did few manual tests and seems that it indeed modifies the initial list of tags. 

I create a quick fix PR, could you have a look ? https://github.com/DataDog/integrations-core/pull/3859

It should not affect the intent of the this function, which is to return the payload object. I tested manually that the change in the PR work. I tried to do a unit test for this, but mocking `vsphere` events is quite tricky and will take more time.